.changeset/brave-pots-drop.md

```diff
@@ -0,0 +1,18 @@
+---
+'astro': minor
+---
+
+Partials
+
+Any page component can now be identified as a *partial*, allowing you fetch them in the client in order to replace only parts of the page. Partials are used in conjuction with a rendering library, like htmx or Stimulus or even just jQuery.
```

Comment:
Since I don't have any comments on the code, I'll just want to mention that as a non-native speaker, I stumbled over not reading a "to" before "fetch". 😅